MerlinAU.sh

```diff
@@ -796,6 +803,14 @@ Update_Custom_Settings()
             then
                 sendEMailNotificationsFlag="$setting_value"
             #
+            elif [ "$setting_type" = "FW_New_Update_EMail_CC_Name" ]
```

Comment:
Approved.

Reply:
Unable to test at this time.

Reply:
Values default values found in file: 

FW_New_Update_Notification_Date TBD
FW_New_Update_Notification_Vers TBD
FW_New_Update_Postponement_Days="0"
FW_New_Update_EMail_Notification="true"

After updating.

Reply:
However:

FW_New_Update_EMail_CC_Name=TBD
FW_New_Update_EMail_CC_Address=TBD

Correctly shows up on a new settings file as expected. I only expect these settings to show up if CC is enabled :)

Reply:
Value is missing on update but exists on new creation. (For now)